packages/account/src/Components/financial-details/financial-details.tsx

```diff
@@ -9,38 +9,27 @@ import {
     Text,
     ThemedScrollbars,
 } from '@deriv/components';
-import { isDesktop, isMobile } from '@deriv/shared';
+import { isDesktop, isMobile, EMPLOYMENT_VALUES } from '@deriv/shared';
```

Comment:
This is intentionally kept as functions for now to avoid path changes in `account-wizard-form.js` since StoreProvider ref cannot be passed from `accounts` package to `core`. Will replace this ina another refactoring card